CardData.py

```diff
@@ -92,93 +96,91 @@ def create_card_json(gwent_data_helper, patch):
     for template_id in card_templates:
         template = card_templates[template_id]
         card = {}
-        card['ingameId'] = template.attrib['id']
-        card['strength'] = int(template.attrib['power'])
-        card['type'] = template.attrib['group']
-        card['faction'] = template.attrib['factionId'].replace("NorthernKingdom", "Northern Realms")
-
-        key = template.attrib['dbgStr'].lower().replace(" ", "_").replace("'", "")
-        # Remove any underscores from the end.
-        if key[-1] == "_":
-            key = key[:-1]
+        card_id = template.attrib['Id']
+        card['ingameId'] = card_id
+        card['strength'] = int(template.find('Power').text)
+        card['type'] = TYPES.get(int(template.find('Tier').text))
+        card['faction'] = FACTIONS.get(int(template.find('FactionId').text))
 
         card['name'] = {}
         card['flavor'] = {}
         for region in GwentUtils.LOCALES:
-            card['name'][region] = gwent_data_helper.card_names.get(region).get(key)
-            card['flavor'][region] = gwent_data_helper.flavor_strings.get(region).get(key)
+            card['name'][region] = gwent_data_helper.card_names.get(region).get(card_id)
+            card['flavor'][region] = gwent_data_helper.flavor_strings.get(region).get(card_id)
 
         # False by default, will be set to true if collectible or is a token of a released card.
         card['released'] = False
 
-        if template.find('Tooltip') is not None:
-            tooltip_id = template.find('Tooltip').attrib['key']
-            card['info'] = {}
-            card['infoRaw'] = {}
-            for locale in GwentUtils.LOCALES:
-                tooltip = gwent_data_helper.tooltips[locale].get(tooltip_id)
-                if tooltip is not None:
-                    card['infoRaw'][locale] = tooltip
-                    card['info'][locale] = GwentUtils.clean_html(tooltip)
+        card['info'] = {}
+        card['infoRaw'] = {}
+        for locale in GwentUtils.LOCALES:
+            tooltip = gwent_data_helper.tooltips[locale].get(card_id)
+            if tooltip is not None:
+                card['infoRaw'][locale] = tooltip
+                card['info'][locale] = GwentUtils.clean_html(tooltip)
 
-            card['keywords'] = gwent_data_helper.keywords.get(tooltip_id)
+        card['keywords'] = gwent_data_helper.keywords.get(card_id)
 
-        card['positions'] = []
+        card['positions'] = ["Melee", "Ranged", "Siege"]
```

Comment:
why default to all positions? (and why vs. `Event`)

Reply:
The concept of `positions` in Gwent doesn't exist anymore. Any unit can be placed on any row. But to support our current data format, we still need to provide the positions.